src/asm/lexer.c

```diff
@@ -921,6 +921,40 @@ void lexer_DumpStringExpansions(void)
 	free(stack);
 }
 
+/* Discards an block comment */
+static void discardBlockComment(void)
+{
+	dbgPrint("Discarding block comment\n");
+	bool block_ended = false;
+
+	for (;;) {
+		switch(nextChar()) {
+			case EOF:
+				error("Unterminated block comment\n");
+				return;	
+			case '/':
+				if (peek(0) == '*') {
+					warning(WARNING_NESTED_COMMENT,
+						"/* in block comment\n");
+					continue;
+				}
+				/* fallthrough */
```

Comment:
This is buggy, as it will cause `//` to end comments. It would make sense if the `peek(0)` was a `nextChar()`, though.